examples/Receiver_Response.py

```diff
@@ -147,6 +147,11 @@ def get_freqs(thisfile):
     # Convolve using $\lambda_{G}$ specified above
     rec_data.convolve("t", lambda_G, enforce_causality=False)
     # }}}
+    # {{{ center data at 0 MHz thus converting to
+    #     $\Delta\nu$ rather than $\nu_{test}$
+    rec_data["nu_test"] -= carrier
+    rec_data.rename("nu_test", Dnu_name)
+    # }}}
```

Comment:
This way the x axis for the 2D and 1D are both relative/consistent and make more sense together

Reply:
also helps that the y axis (direct) is relative to the carrier so that when this axis is 0 it means it is outputting at the carrier